HLTriggerOffline/JetMET/python/Validation/SingleJetValidation_cfi.py

```diff
@@ -20,4 +20,9 @@
     HLTriggerResults      = cms.InputTag("TriggerResults::HLT"),
 )
 
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+phase2_common.toModify(SingleJetMetPaths,
+                       PatternJetTrg = cms.untracked.string("HLT_(AK4)?PF(NoPU|Puppi)?Jet([0-9])+(_v[0-9]+)?$")
+                       )
+
```

Comment:
Many of the new histograms are empty. I think the reason (or at least, one reason) is that the value of `PFJetAlgorithm` should be updated to `hltAK4PFPuppiJetsCorrected` in phase-2 wfs.

Reply:
Part of the empty histogram is GenJet collection.